viam_cartographer.go

```diff
@@ -337,16 +341,24 @@ func parseCartoAlgoConfig(configParams map[string]string, logger logging.Logger)
 			}
 			cartoAlgoCfg.MissingDataRayLength = float32(fVal)
 		case "max_range":
+
```

Comment:
Mild preference to remove this line again, and never start a new indentation level with a blank line.